packages/inputs-minimal/package.json

```diff
@@ -29,10 +29,7 @@
     "lint:typescript": "../../scripts/check-typescript"
   },
   "peerDependencies": {
-    "@babel/runtime": "^7.11.2",
```

Comment:
you can't just change peer dependency versions. every change in peer dep versions are breaking changes and therefore need a major version bumb (not worth in this case i guess)

https://wiki.nebenan.de/display/AWSM/1st+Party+Packages#id-1stPartyPackages-Peerdependencies